pylint_odoo/checkers/no_modules.py

```diff
@@ -430,10 +439,10 @@ def get_decorators_names(self, decorators):
     @utils.check_messages('translation-required')
     def visit_raise(self, node):
         args = misc.join_node_args_kwargs(node.last_child())
-        if args:
-            for argument in args:
-                if isinstance(argument, astroid.Const) and \
-                        argument.name == 'str':
-                    self.add_message('translation-required', node=node,
-                                     args=(node.last_child().func.name,
-                                           argument.as_string()))
+        for argument in args:
+            if isinstance(argument, astroid.Const) and \
+                    argument.name == 'str':
+                self.add_message('translation-required', node=node,
+                                 args=(self.get_func_name(
```

Comment:
@moylop260 
that atttribute doesn't have the func node, I try with that attribute and doesn't work